data/remnant/remnant 2 side missions.txt

```diff
@@ -715,3 +715,58 @@ mission "Remnant: Scent of Rain 1"
+	description "Travel to the <waypoints> system to fight and disable a stranded Kas'lor Ik 582. Do not destroy or capture the ship, as it likely has a large civilian population."
+	source
+		government "Remnant"
+		not attributes "station"
+	waypoint "Antevorta"
+	to offer
+		and
+			"remnant: broken jump drive count" >= 10
+			or
+				and
+					has "Remnant: Cognizance 25: done"
+					not "flagship planet: Ssil Vida"
+				has "event: remnant: cognizance calm"
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a difficult request for you."`
+			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Kas'lor Ik 582 to be precise. Normally we would request you track down this ship and eliminate it, but because ships of this type tend to be home to a large civilian population, destroying or capturing it outright may instigate a reprisal, especially in that sector."`
+			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can disarm the ship and fix their jump drive and hope they decide to head back home."`
+			choice
+				`	"Happy to help those in need, even the Korath."`
+				`	"I have no intentions of risking my neck for some Korath."`
+					goto rathernot
+			`	"I am grateful for your understanding and willingness to help here. Let's make sure everyone gets home safely."`
+				accept
+			label rathernot
+			`	He nods in acknowledgment. "I feared my ask was too much. I more than understand the sentiment. Maybe I was being naive to think such an outcome was possible."`
+				decline
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system "Antevorta"
+		ship "Kas'lor Ik 582 (Crippled)" "Flumgorkor"
+		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
+	on enter "Antevorta"
+		dialog
+			`As you enter the system your scanners pick the target ship. Your nerves are acting up at the thought of having to take on such a powerful ship, but you remind yourself to think of civilians on board, possibly even children.`
+	on abort
+		"reputation: Remnant" += 250
+	on fail
+		"reputation: Remnant" -= 250
+		dialog
+			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
+	on complete
+		"remnant: disable and save count" ++
+		payment 2500000
+		conversation
+			`You find Chilia with a relieved look on his face when you land.`
+			`	"This mission wasn't my idea. Taking on the larger Korath vessels is a sensitive matter. You may frequently find large groups of Korath children. I must admit I was worried you wouldn't be those children's best chance, but our fleet was too tied up to take this one at the moment. I am glad I was not wrong." he says with a content smile on his face.`
```

Comment:
```suggestion
			`	"These assignments are typically handled by our most trusted teams, as taking on the larger Korath vessels is a sensitive matte. Most of their larger ships are home to large groups of Korath civilians, including children. I must admit there were concerns you might not be those children's best chance, but our fleet was occupied elsewhere at the time. I am glad I was not wrong in my assessment of you." he sings with a content smile on his face.`
```

It is subtle, but shifting the concerns from Chilia specifically to the Remnant in general is intentional to suggest and reinforce the idea that there is a significant collaborative decision making process behind the scenes.